File: update.py
import json
with open('/workspace/client/package.json', 'r') as f:
    package = json.load(f)
package['scripts']['coverage:cleanup'] = 'node scripts/cleanup-coverage.js'
with open('/workspace/client/package.json', 'w') as f:
    json.dump(package, f, indent=4)
    f.write('\n')

File: update.js
const fs = require("fs");
const pkg = JSON.parse(fs.readFileSync("/workspace/client/package.json", "utf8"));
pkg.scripts["coverage:cleanup"] = "node scripts/cleanup-coverage.js";
fs.writeFileSync("/workspace/client/package.json", JSON.stringify(pkg, null, 4) + "\n");
